mypy/applytype.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
-from typing import Callable, Iterable, Sequence
+from collections.abc import Iterable, Sequence
+from typing import Callable
```

Comment:
`collections.abc.Callable` is broken inside `Union` and `Optional` on `3.9.0` and `3.9.1`, so it isn't safe to replace just yet. https://bugs.python.org/issue42965

_We could consider bumping `requires-python` to `>=3.9.2` or just wait until we drop 3.9 next year._